lib/jsdom/living/helpers/colors.js

```diff
@@ -0,0 +1,185 @@
+"use strict";
+
+exports.computeColor = maybeColor => {
+  maybeColor = maybeColor.trim();
+
+  // sRGB?
+  const srgbRegex = /^rgb *\( *(\d{1,3}) *, *(\d{1,3}) *, *(\d{1,3}) *\)$/;
+  if (srgbRegex.test(maybeColor)) {
+    const [, r, g, b] = srgbRegex.exec(maybeColor);
+    return `rgb(${r}, ${g}, ${b})`;
+  }
+
+  // hex?
+  const shorthandHexRegex = /^#([a-f\d])([a-f\d])([a-f\d])$/;
+  maybeColor = maybeColor.replace(shorthandHexRegex, (_, r, g, b) => {
+    return r + r + g + g + b + b;
+  });
+  const hexRegex = /^#([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/;
+  if (hexRegex.test(maybeColor)) {
+    let [, r, g, b] = hexRegex.exec(maybeColor);
+    r = parseInt(r, 16);
+    g = parseInt(g, 16);
+    b = parseInt(b, 16);
+    return `rgb(${r}, ${g}, ${b})`;
+  }
+
+  // definded keyword?
+  if (exports.keywords[maybeColor]) {
+    const [r, g, b] = exports.keywords[maybeColor];
+    return `rgb(${r}, ${g}, ${b})`;
+  }
+
+  return "";
+};
+
+// https://www.w3.org/TR/css-color-3/#svg-color
```

Comment:
Be sure to use Editor's drafts, so in this case https://drafts.csswg.org/css-color-3/. I suspect the list hasn't changed much, but at least the comment should change.